_projects/2021/sunpy/3dplotting.md

```diff
@@ -0,0 +1,51 @@
+---
+name: 3D plotting
+desc: Provide 3D plotting capabilities to SunPy
+# add a short one line description of your project
+requirements:
+# Student requirements:
+ - Familiarity with Python
```

Comment:
Maybe consider something about having at least tried some plotting code in python whether it be 2d or 3d?